aboutMePage/aboutMePage.php

```diff
@@ -0,0 +1,83 @@
+<?php
+require_once '../dbConn.php';
+require_once '../functions.php';
+?>
+
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <link rel="stylesheet" href="../normalize.css" type="text/css" />
+    <title>Joseph:Home</title>
+    <link rel="stylesheet" href="styles.css" type="text/css" />
+</head>
+<body>
+<nav>
+    <div class="navContainer">
+        <div class="logoName">
+            JaHolt
+        </div>
+        <ul>
+            <a href="../contactPage/contactPage.php"><li>CONTACT</li></a>
```

Comment:
You shouldn't be putting a tags around entire elements. put the a tag inside the <li> element